CORE_CONTRIBUTOR.md

```diff
@@ -11,7 +11,8 @@ Contributors who have displayed lasting commitment to the evolution and maintena
 
 
 ## As core contributors, we:
-- Review and merge pull requests
+- Review pull requests
+- Merge pull requests we review, except for PRs where the author has commit access. Commit access is noted in Github with the `Member` tag
```

Comment:
I believe the term GitHub uses is "Push access" rather than "Commit access"?

Maybe describe the reason from the intent (= "don't merge for them… because they have the necessary rights to merge it themselves") instead of the consequence (commit access)?

> Merge pull requests we review, except for PRs where the author themselves has the right to merge them. Github annotate those authors with the `Member` tag